contracts/Enforcer.sol

```diff
@@ -102,6 +101,6 @@ contract Enforcer {
         }
 
         bool success = address(0).send(bondAmount);
-        require(success == true);
+        require(success == true, "Cannot burn");
```

Comment:
I don't know how to fail this line.

Reply:
😆 Ya that is a ugly one. This part I deal with in https://github.com/leapdao/solEVM-enforcer/pull/66/files

Reply:
Nice, I'll leave it alone for now.